src/app/api/user/[id]/unsafemetadata/route.ts

```diff
@@ -0,0 +1,27 @@
+import { NextResponse, NextRequest } from "next/server";
+import { clerkClient } from "@clerk/nextjs";
+import connectDB from "@/database/db";
+import Organization from "@/database/organizationSchema";
+
+type IParams = {
+  params: {
+    id: string;
+  };
+};
+
+export async function GET(req: NextRequest, { params }: IParams) {
+  await connectDB();
+  const { id } = params;
+  try {
+    const organization = await Organization.findOne({ clerkUser: id });
```

Comment:
my bad for not making this clear: we will no longer be using the organization db so you can pretend that doesn't exist anymore. instead we will be storing everything in the organization schema inside `unsafeMetadata` so that `User` and `Organization` will essentially become the same thing. since User and Organization have a one-to-one relationship, this removes the need for any linking logic like you did here and makes user/organization management much simpler.

so the structure of our project will look like this:
client:
- get/set user info (via signed in user) -> clerk as backend
  - clerk user contains an array of reimbursement ids
- get/set reimbursement info (via reimbursement id) -> our API as backend

and just for context, the reason we are combining user and organization into a single object is because the client told as that each organization will only have one rep (most likely the treasurer) working in our portal, making it one-to-one.